package.json

```diff
@@ -5,7 +5,10 @@
   "main": "src/ImageSequencer.js",
   "scripts": {
     "debug": "TEST=true node ./index.js -i ./examples/images/monarch.png -s invert",
-    "test": "TEST=true istanbul cover tape test/core/*.js test/core/ui/user-interface.js test/core/modules/*.js | tap-spec; node test/core/sequencer/benchmark.js | tap-spec; node test/core/gifs/gif-test.js | tap-spec; grunt tests; cat ./output/core-tests.js | tape-run --render=\"tap-spec\"",
+    "test": "TEST=true istanbul cover tape test/core/*.js test/core/ui/user-interface.js test/core/modules/*.js | tap-spec;",
```

Comment:
Can we add a more script to run all tests!

Reply:
There is one issue in that, for example I want to add a GIF test in my "tests suite"... GIF's uses puppeteer to work, but istanbul doesn't work with puppeteer, If we pull all the test in one js, then this will create issues. I tried it when writting test for gif. If we remove istanbul then we dont get coverage report, If we remove puppeteer we wont be able to work gif's.

Reply:
how about
```js
"test-all": "npm run test && npm run benchmark && npm run gif-test && npm run core-tests"
```
Would this work?

Reply:
okay..... now I got what you meant...Yeah that will work, will do the changes.